crypto/rand/rand_vms.c

```diff
@@ -497,6 +502,72 @@ int rand_pool_add_nonce_data(RAND_POOL *pool)
     return rand_pool_add(pool, (unsigned char *)&data, sizeof(data), 0);
 }
 
+/*
+ * SYS$GET_ENTROPY METHOD
+ * ======================
+ *
+ * This is a high entropy method based on a new systems service that is
+ * based on getentropy() from FreeBSD 12.  It's only used if available,
```

Comment:
based on the getentropy() system call

Reply:
Whether `getentropy()` is a system call or a library function seems to vary, even between BSD branches, so let's avoid that qualifier, yeah?

Reply:
okidoki